src/DynamoCoreWpf/Views/Core/DynamoView.xaml

```diff
@@ -609,19 +609,13 @@
                               Name="PackageManagerMenu"
                               Focusable="False"
                               IsEnabled="True">
-                        <MenuItem Header="{x:Static p:Resources.DynamoViewPackageMenuSearchPackage}"
-                                  Command="{Binding Path=DataContext.ShowPackageManagerSearchCommand, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type controls:DynamoView}}}"
-                                  Name="showPMSearch" />
-                        <MenuItem Header="{x:Static p:Resources.DynamoViewPackageMenuPublishPackage}"
-                                  Command="{Binding Path=DataContext.PublishNewPackageCommand, RelativeSource={RelativeSource FindAncestor, AncestorType={x:Type controls:DynamoView}}}"
-                                  Name="publishNewPackage" />
```

Comment:
If we are removing the menu items, lets' remove the resources as well

Reply:
OK, will do, yes!